packages/tupaia-web/src/api/queries/useLandingPage.ts

```diff
@@ -5,14 +5,13 @@
  */
 
 import { useQuery } from 'react-query';
-import { useLocation } from 'react-router';
+import { useParams } from 'react-router';
 import { get } from '../api';
 import { SingleLandingPage } from '../../types';
 
-export const useLandingPage = (urlSegment?: string) => {
-  const { pathname } = useLocation();
-  // occasionally the urlSegment is not passed in, so we need to get it from the pathname. This is in cases where this hook is accessed from a modal, for example, that doesn't have access to the urlSegment param
-  const landingPageUrlSegment = urlSegment || pathname.split('/')[1];
+export const useLandingPage = () => {
+  const { landingPageUrlSegment } = useParams();
```

Comment:
I ended up just combining the url param into the api hook since the usLanding page check is based on the url anyway